Source/WebKit/UIProcess/mac/WebViewImpl.h

```diff
@@ -696,6 +696,11 @@ ALLOW_DEPRECATED_DECLARATIONS_END
     void beginTextRecognitionForVideoInElementFullscreen(ShareableBitmap::Handle&&, WebCore::FloatRect);
     void cancelTextRecognitionForVideoInElementFullscreen();
 
+#if HAVE(INLINE_PREDICTIONS)
+    void setInlinePredictionsEnabled(bool);
```

Comment:
Not entirely clear why these are out of line (but, also fine...)

Reply:
good point. will fix